app/bundles/LeadBundle/Entity/LeadEventLogRepository.php

```diff
@@ -36,6 +36,23 @@ public function getFailedRows($importId, array $args = [], $bundle = 'lead', $ob
         return $this->getSpecificRows($importId, 'failed', $args, $bundle, $object);
     }
 
+    public function getEntities(array $args = [])
+    {
+        $entities = parent::getEntities($args);
+        $entities = iterator_to_array($entities);
```

Comment:
@alanhartless could you chime in whether this isn't a BC break? The `getEntities` used to return the Iterator and now it will return an array.

@mitresh95 unit tests for the special error with the SQL would be appreciated.

Reply:
@escopecz I think it by default returns a Paginator but yes, it technically could be a BC break. And if we wanted to bypass using Paginator, we should set `$args['ignore_paginator'] = true` to prevent multiple transformations. But @mitresh95 was this changed? I thought originally it was in a template.